core/config/project_settings.cpp

```diff
@@ -1473,7 +1473,7 @@ ProjectSettings::ProjectSettings() {
 	GLOBAL_DEF(PropertyInfo(Variant::INT, "display/window/size/window_height_override", PROPERTY_HINT_RANGE, "0,4320,1,or_greater"), 0); // 8K resolution
 
 	GLOBAL_DEF("display/window/energy_saving/keep_screen_on", true);
-	GLOBAL_DEF("display/window/energy_saving/keep_screen_on.editor", false);
+	GLOBAL_DEF("display/window/energy_saving/keep_screen_on.editor_hint", false);
```

Comment:
Should this also be enclosed in `#ifdef TOOLS_ENABLED`?

Reply:
It could.